lib/propolis/src/hw/nvme/mod.rs

```diff
@@ -154,6 +182,10 @@ const MAX_NUM_IO_QUEUES: usize = MAX_NUM_QUEUES - 1;
 
 /// NVMe Controller
 struct NvmeCtrl {
+    /// A distinguishing identifier for this NVMe controller across the VM.
+    /// Useful mostly to distinguish queues and commands as seen in probes.
```

Comment:
```suggestion
    /// Useful mostly to distinguish queues and commands for parameters to DTrace USDT probes.
```

...more keyword-y for future lazy grepping?